MANIFEST.in

```diff
@@ -14,6 +14,7 @@ recursive-include docs *
 recursive-include res *
 recursive-include scripts *
 recursive-include scripts-dev *
+recursive-include synapse *.pyi
```

Comment:
I think the "source distribution" is for distribution for running, rather than distribution for editing. But we can leave it in I suppose